pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -129,44 +110,44 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 		}))
 		if updateErr != nil {
 			c.eventRecorder.Warning("EtcdMembersErrorUpdatingStatus", updateErr.Error())
-			return err
+			return updateErr
```

Comment:
don't return these errors, just keep going.

Reply:
you may aggregate them if you wish, but set everything

Reply:
done